### PR TITLE
added link to and explanation of the test generator to the contributi…

### LINKED ISSUE #185: Generator for tests
Hi all,
## I'm currently developing a Ruby and ERB based generator to make the test files for our problems based on a template and the data from problem specifications.  I would say I'm in middle alpha stage.
1. Successfully generates tests when everything is in the same directory
2. Successfully generates tests when the template is in another file.
#### Currently I'm ironing out a couple things to make it friendlier to contributors who will use this
- In the end I plan on using a shell wrapper for the ruby program
- I will need to generate in depth documentation so that implementation of generator templates for exercises can be done by any contributor.
- to simplify the usage and creation of such tools I think we should create the following **conventions**:
  - scripts that involve this repo and other repos extended from exercism should expect all the repos to be in parallel directories.  This is expected in the ruby track and specified in their [README.md](https://github.com/guygastineau/ruby/blob/master/README.md) in the *regenerating a test suite* section.
    - This would allow scripts in the bash track to be shorter and more straight forward.
  - Documentation of developement tools should be comprehensive - especially if the tool is not written exclusively in bash.
    - This generator project will be mostly ruby and I think it is appropriate for documentation on it to make it as accessible as possible even to those unfamiliar with ruby.

### PR DESCRIPTION
**Pull request by guygastineau**
…ng doc

I went ahead and removed the newline in our example and template of the tests as this does not seem idiomatic in BATS compared to the format on the main repo [here](
https://github.com/sstephenson/bats/blob/master/README.md).

closes #185 

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

**Comment by guygastineau** [comment]
@budmc29 

I can add a line explicitly stating that usage of this tool is not mandatory if you want.
I'm not sure if that is clear or needs to be defined outright.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -149,6 +148,15 @@ You can check to see if your example script works by running `bats`.
 ```bash
 $ bats hello_world_test.sh
 ```
+#### Bash-Track-Test-Generator
+
+There is a Ruby and ERB based automated solution for generating tests
+at [guygastineau/bash-track-test-generator](https://github.com/guygastineau/bash-track-test-generator).
+
+Each exercise requires an erb template to generate the test, and there is a
```
[the remark this answers is not in the document]
Comment (guygastineau) [reply CONTRIBUTING.md]: Okay. Thanks for the feedback. I'll be at work for the next couple of hours but I'll come up with some edits when I get home.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -149,6 +148,15 @@ You can check to see if your example script works by running `bats`.
 ```bash
 $ bats hello_world_test.sh
 ```
+#### Bash-Track-Test-Generator
+
+There is a Ruby and ERB based automated solution for generating tests
```
[the remark this answers is not in the document]
Comment (guygastineau) [reply CONTRIBUTING.md]: You think I should add that it does this in the description?

[the remark this answers is not in the document]
Comment (budmc29) [reply CONTRIBUTING.md]: Yes, one extra reason for someone to use the tool. In my opinion spotting the differences is boring, it's good to have a tool that does that.

**Comment by guygastineau** [comment]
Let me know what you think about the changes I made.  I think it is clear that it is optional so I didn't state that explicitly.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -148,13 +148,16 @@ You can check to see if your example script works by running `bats`.
 ```bash
 $ bats hello_world_test.sh
 ```
-#### Bash-Track-Test-Generator
+#### Automatic test generator
+
+**Tired of reading JSON by hand everytime an exercise version is updated?**
```
Comment (budmc29) [inline CONTRIBUTING.md]: Sounds like a beginning to an ad, but it's funny so it's acceptable 😄

**Comment by budmc29** [comment]
Thank you.

**Pull request merged**